BeagleBone/AI/pru/prugpio.h

```diff
@@ -0,0 +1,67 @@
+// This file defines the GPIO port addresses and PRU address
```

Comment:
Is this generic enough to be part of pru-support-package?

Reply:
It could be, but right now it very incomplete.